src/clr-core/import-map.importmap

```diff
@@ -0,0 +1,29 @@
+  
+{
+  "imports": {
+    "tslib": "/base/node_modules/tslib/tslib.es6.js",
```

Comment:
This file tells chrome how to load the module natively. I think in the near future I will be able to automate this part using some existing 3rd party tools but for now its manual. This will also set us up to easily support pure JS/HTML/CDN setups with no build step.

Reply:
Will the build process fail if we miss a 3r party import here? I can already see that I'll need to add a couple for the modal PR, for example.

Reply:
Yeah this will fail during the build if a file is missing and not listed here. Will be automated away in the near future.